src/MaticeServiceProvider.php

```diff
@@ -19,30 +17,12 @@ public function boot()
         /*
          * Optional methods to load your package assets
          */
-        // $this->loadTranslationsFrom(__DIR__.'/../resources/lang', 'matice');
-        // $this->loadViewsFrom(__DIR__.'/../resources/views', 'matice');
-        // $this->loadMigrationsFrom(__DIR__.'/../database/migrations');
-        // $this->loadRoutesFrom(__DIR__.'/routes.php');
-
```

Comment:
They were mainly there for documenting purpose. Maybe we can keep them so that future PR makers know them easily.